### DISCUSSION TITLE
Have NOTE categories?

### DISCUSSION THREAD

**Discussion by mother10** [FamilySearch/GEDCOM#497] [Ideas]
I searched the discussions if anything like this had come up, but could not find it.

For years I have been struggling with a way of using NOTE's the way I want them to be.
For now, GEDCOM only has 1 kind of NOTE's.

Inside it you can put remarks, but what kind of?
- It can be remarks for a person, telling something interesting about that person. Those are remarks you want printed in reports.
- Or, it can be remarks you need to be able to further investigate something later in time. You dont want these on reports!
- Or it can be remarks you want to stay private, for whatever reason. these should only be printed under certain conditions.
- Or it can be a Book chapter kind of Note. Containing longer texts that can be useed to print some kind of Book from (part of) your tree.

As it is now you have to struggle with all kind of extra **_TAGS**, to get the result you want (if your software allows that). And sometimes you just have to face a lot of rubbish printed.

What I do now is put all kind of info in a note, that help me investigate more about something, later in time. But either i use some (illegal) private GEDCOM tag the software I use, offers me, or I get all kind of rubbish printed in my reports.

I would very much like a decent way to deal with this.
So for instance, have NOTE's for the real info I want in my reports.
Have some kind of "Scratch NOTE or Investigate NOTE", for the info that should not be printed in reports, but only in special separate investigation lists.
And have a normal kind of NOTE, where I can put private information, that should only be printed in special reports, like for my own family.

As said, I struggle with this for many years. My old program "invented" invisible text by putting a * before and after the tekst.
Another program used [ and ] to denote invisible text inside a note.  

Also, there are many users asking how to enter "HTML"like, or "CSS"like structures inside a NOTE, to be able to format the text, with colors, fontsize and more.

I have a nephew who added very long NOTE's to be able to create a book for his family. But for those who just wanted a treeview of some kind, that extra text would only clutter their view.

I propose the following:

NOTE - for backward compatability. Handled by software as they do it until now.
NOTEP - Private note's. Everything in here will only be printed according to privacy settings. So just for family, not for the whole world.
NOTES - Scratchbook, investigation Note's. Information in these is only used by the owner of the tree to investigate something further. More like a real scratchbook so to speak.
NOTEB - Book kind of notes, like for what my nephew did. Should only be printed in "Book" kind of reports.

Each of these 4 could be formatted or not, so maybe we need a TYPE for formatting.

Each of these 4 note's could go, where it is legal now to have a Note_Structure.

Maybe its easier to have a NOTE_TYPE  with: NORMAL, PRIVATE, SCRATCH, BOOK, FORMATTED
And add that to the Note. But in that case combinations like PRIVATE, FORMATTED should be possible.

Any ideas if something like this would be possible?

**Comment by tychonievich** [comment]
Discussed in steering committee 2 JUL 2024

This topic has been discussed before in #219; we recommend the comments there as a better place for this discussion.
